nodes/vr_sros/vr-sros.go

```diff
@@ -129,3 +166,8 @@ func createVrSROSFiles(node nodes.Node) error {
 
 	return nil
 }
+
+// isPartialConfigFile returns true if the provided config is a partial config
+func isPartialConfigFile(c string) bool {
+	return strings.Contains(c, ".partial.")
```

Comment:
I'd enforce capitalization and checking against `.PARTIAL` without a trailing dot, to make sure that `sr.cfg.partial` and `sr.partial.cfg` would be both accepted

Reply:
really capital? thats files screaming at me ... lets do a strings.Lower(c) and we're fine

Reply:
I mean you capitalize the string in the code to allow both lowercase and uppercase to be passed by a user.